android/PhysicalWeb/app/src/main/res/layout/list_item_nearby_beacon.xml

```diff
@@ -21,6 +21,22 @@
                 android:layout_width="0dp"
                 android:layout_height="wrap_content"
                 android:orientation="vertical"
+                android:layout_weight="1"
+                android:gravity="start"
+                >
```

Comment:
lgtm, but could you move the closing angle bracket back to the android:gravity line?